WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -53,3 +53,13 @@ extension UIAlertController {
         copyAlertController?.presentFromRootViewController()
     }
 }
+
+@objc extension UIAlertController {
```

Comment:
This feels like overloading ReachabilityUtils since this notices can be displayed on any error. Granted most of the time this error would be due to a bad/ no connection so I'm willing to use that but we should figure out another way to use the notices in objc. Maybe outside the scope of this

Reply:
This feels like overloading ReachabilityUtils since this notices can be displayed on any error. Granted most of the time this error would be due to a bad/ no connection so I'm willing to use that but we should figure out another way to use the notices in objc. Maybe outside the scope of this

Reply:
> Granted most of the time this error would be due to a bad/ no connection so I'm willing to use that but we should figure out another way to use the notices in objc.

Yes, that's why I was suggesting to use `Reachability`. Regarding Obj-C, I wish we could convert the call sites to Swift instead. 😄 But it's definitely not trivial.

Reply:
>Yes, that's why I was suggesting to use Reachability

@shiki ,not sure I understand you here. You're suggesting to use ReachabilityUtils to show a notice with  showNoInternetConnectionNoticeWithMessage regardless if it's because of no internet connection. 
For example, I'd like to display a notice when the reply was successfully sent, instead of a HUD. Should I use showNoInternetConnectionNoticeWithMessage  in that case too?

Reply:
> Should I use showNoInternetConnectionNoticeWithMessage in that case too?

Good point! I have forgotten that we cannot use Notices in Obj-C. 

For the error messages, let's still use `ReachabilityUtils`, not the `UIAlertController` extensions. As for the HUD, we can figure that out until we have a better solution for using Notices in Obj-C.

Reply:
I actually think I’m gonna to use Nate’s suggestion: https://github.com/wordpress-mobile/WordPress-iOS/pull/11441#issuecomment-481795399
And add those methods in UIViewController+Notice is you have no objection

Reply:
No objection! 🙂